### DISCUSSION TITLE
Работа скриптов на manjaro

### DISCUSSION THREAD

**Discussion by Simper007** [ffcrus/garlic#6] [General]
На manjaro 23 скрипты работают. Звук настроил, тоже всё корректно. Огромное спасибо автору! Незаменимая штука!

**Comment by Artemsites** [comment]
Thanks a lot to the author, kde works on manjaro!
but the script layout_changing_previous.sh sometimes it inserts from the clipboard

the main thing is not to forget to install dependencies: 

```
pamac install xdotool xsel sed
```
